packages/mds-agency/tests/integration-tests.ts

```diff
@@ -142,30 +138,6 @@ after(async () => {
 })
 
 describe('Tests API', () => {
-  it('verifies non-uuid provider_id is rejected', done => {
-    request
-      .get('/devices')
-      .set('Authorization', AUTH_GARBAGE_PROVIDER)
-      .expect(400)
-      .end((err, result) => {
-        test.value(result).hasHeader('content-type', APP_JSON)
-        test.string(result.body.result).contains('invalid provider_id', 'is not a UUID')
-        done(err)
-      })
-  })
-
-  it('verifies unknown provider_id is rejected', done => {
-    request
-      .get('/devices')
-      .set('Authorization', AUTH_UNKNOWN_UUID_PROVIDER)
-      .expect(400)
-      .end((err, result) => {
-        test.value(result).hasHeader('content-type', APP_JSON)
-        test.string(result.body.result).contains('invalid provider_id', 'is not a known provider')
-        done(err)
-      })
-  })
-
```

Comment:
Why are we removing these tests?

Reply:
These endpoints simply don't exist in the API.

Reply:
The reason they pass at all is because the auth token is bad.

Reply:
Right right, cool 👍